### PR TITLE
Add load test with and without cache

### PR DESCRIPTION
**Pull request by kalaiyarasiganeshalingam**
## Purpose
$Subject

Fixes: https://github.com/ballerina-platform/ballerina-standard-library/issues/2461

Test result for local:

**Non-cache**

```
summary +    579 in 00:00:05 =  122.4/s Avg:    21 Min:     3 Max:   179 Err:     0 (0.00%) Active: 5 Started: 5 Finished: 0
summary +  15526 in 00:00:30 =  517.7/s Avg:    38 Min:     1 Max:   459 Err:     0 (0.00%) Active: 35 Started: 35 Finished: 0
summary =  16105 in 00:00:35 =  463.8/s Avg:    37 Min:     1 Max:   459 Err:     0 (0.00%)
summary +  21385 in 00:00:30 =  713.4/s Avg:    64 Min:     1 Max:   263 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary =  37490 in 00:01:05 =  579.5/s Avg:    53 Min:     1 Max:   459 Err:     0 (0.00%)
summary +  39795 in 00:00:30 = 1327.0/s Avg:    45 Min:     0 Max: 30004 Err:    10 (0.03%) Active: 60 Started: 60 Finished: 0
summary =  77285 in 00:01:35 =  816.2/s Avg:    49 Min:     0 Max: 30004 Err:    10 (0.01%)
summary +  41577 in 00:00:30 = 1386.1/s Avg:    43 Min:     0 Max: 30004 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 118862 in 00:02:05 =  953.3/s Avg:    46 Min:     0 Max: 30004 Err:    20 (0.02%)
summary +  40323 in 00:00:30 = 1343.5/s Avg:    45 Min:     0 Max: 30004 Err:    11 (0.03%) Active: 60 Started: 60 Finished: 0
summary = 159185 in 00:02:35 = 1029.0/s Avg:    46 Min:     0 Max: 30004 Err:    31 (0.02%)
summary +  40020 in 00:00:30 = 1334.1/s Avg:    44 Min:     0 Max: 30004 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 199205 in 00:03:05 = 1078.6/s Avg:    46 Min:     0 Max: 30004 Err:    41 (0.02%)
summary +  42125 in 00:00:30 = 1404.7/s Avg:    42 Min:     0 Max: 30004 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 241330 in 00:03:35 = 1124.1/s Avg:    45 Min:     0 Max: 30004 Err:    51 (0.02%)
summary +  44712 in 00:00:30 = 1490.4/s Avg:    40 Min:     0 Max: 30004 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 286042 in 00:04:05 = 1169.0/s Avg:    44 Min:     0 Max: 30004 Err:    61 (0.02%)
summary +  45414 in 00:00:30 = 1513.9/s Avg:    39 Min:     0 Max: 30002 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 331456 in 00:04:35 = 1206.7/s Avg:    43 Min:     0 Max: 30004 Err:    71 (0.02%)
summary +  42651 in 00:00:30 = 1421.2/s Avg:    42 Min:     0 Max: 30003 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 374107 in 00:05:05 = 1227.8/s Avg:    43 Min:     0 Max: 30004 Err:    81 (0.02%)
summary +  41667 in 00:00:30 = 1389.4/s Avg:    43 Min:     0 Max: 30002 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 415774 in 00:05:35 = 1242.3/s Avg:    43 Min:     0 Max: 30004 Err:    91 (0.02%)
summary +  41905 in 00:00:30 = 1396.9/s Avg:    42 Min:     0 Max: 30003 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 457679 in 00:06:05 = 1255.0/s Avg:    43 Min:     0 Max: 30004 Err:   101 (0.02%)
summary +  38066 in 00:00:30 = 1268.8/s Avg:    47 Min:     0 Max: 30003 Err:    10 (0.03%) Active: 60 Started: 60 Finished: 0
summary = 495745 in 00:06:35 = 1256.0/s Avg:    43 Min:     0 Max: 30004 Err:   111 (0.02%)
summary +  38863 in 00:00:30 = 1295.5/s Avg:    46 Min:     1 Max: 30004 Err:    10 (0.03%) Active: 60 Started: 60 Finished: 0
summary = 534608 in 00:07:05 = 1258.8/s Avg:    44 Min:     0 Max: 30004 Err:   121 (0.02%)
summary +  40779 in 00:00:30 = 1359.3/s Avg:    44 Min:     0 Max: 30003 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 575387 in 00:07:35 = 1265.5/s Avg:    44 Min:     0 Max: 30004 Err:   131 (0.02%)
summary +  39549 in 00:00:30 = 1318.3/s Avg:    45 Min:     0 Max: 30004 Err:    10 (0.03%) Active: 60 Started: 60 Finished: 0
summary = 614936 in 00:08:05 = 1268.7/s Avg:    44 Min:     0 Max: 30004 Err:   141 (0.02%)
summary +  39102 in 00:00:30 = 1303.4/s Avg:    45 Min:     0 Max: 30003 Err:    10 (0.03%) Active: 60 Started: 60 Finished: 0
summary = 654038 in 00:08:35 = 1270.8/s Avg:    44 Min:     0 Max: 30004 Err:   151 (0.02%)
summary +  41561 in 00:00:30 = 1385.2/s Avg:    43 Min:     0 Max: 30003 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 695599 in 00:09:05 = 1277.1/s Avg:    44 Min:     0 Max: 30004 Err:   161 (0.02%)
summary +  40975 in 00:00:30 = 1366.0/s Avg:    43 Min:     0 Max: 30003 Err:    10 (0.02%) Active: 60 Started: 60 Finished: 0
summary = 736574 in 00:09:35 = 1281.7/s Avg:    44 Min:     0 Max: 30004 Err:   171 (0.02%)
summary +  33807 in 00:00:33 = 1026.7/s Avg:    47 Min:     0 Max: 30005 Err:    11 (0.03%) Active: 9 Started: 60 Finished: 51
summary = 770381 in 00:10:08 = 1267.9/s Avg:    44 Min:     0 Max: 30005 Err:   182 (0.02%)
summary +      8 in 00:00:22 =    0.4/s Avg: 30002 Min: 30000 Max: 30004 Err:     8 (100.00%) Active: 0 Started: 60 Finished: 60
summary = 770389 in 00:10:30 = 1223.6/s Avg:    44 Min:     0 Max: 30005 Err:   190 (0.02%)

```

**Cache with cleanup task**

```
summary +  12674 in 00:00:18 =  688.5/s Avg:    13 Min:     1 Max:   199 Err:     0 (0.00%) Active: 19 Started: 19 Finished: 0
summary +  52369 in 00:00:30 = 1745.7/s Avg:    19 Min:     0 Max:   112 Err:     0 (0.00%) Active: 49 Started: 49 Finished: 0
summary =  65043 in 00:00:48 = 1343.7/s Avg:    18 Min:     0 Max:   199 Err:     0 (0.00%)
summary +  99981 in 00:00:30 = 3332.8/s Avg:    17 Min:     0 Max:   111 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 165024 in 00:01:18 = 2104.8/s Avg:    17 Min:     0 Max:   199 Err:     0 (0.00%)
summary +  98357 in 00:00:30 = 3278.6/s Avg:    18 Min:     0 Max:   148 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 263381 in 00:01:48 = 2429.6/s Avg:    17 Min:     0 Max:   199 Err:     0 (0.00%)
summary +  95005 in 00:00:30 = 3166.8/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 358386 in 00:02:18 = 2589.4/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  97645 in 00:00:30 = 3254.1/s Avg:    18 Min:     0 Max:   103 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 456031 in 00:02:48 = 2707.8/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary + 100917 in 00:00:30 = 3364.7/s Avg:    17 Min:     0 Max:   102 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 556948 in 00:03:18 = 2807.1/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  99736 in 00:00:30 = 3324.5/s Avg:    18 Min:     0 Max:    97 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 656684 in 00:03:48 = 2875.1/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  95968 in 00:00:30 = 3198.4/s Avg:    18 Min:     0 Max:   136 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 752652 in 00:04:18 = 2912.6/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  98529 in 00:00:30 = 3284.8/s Avg:    18 Min:     0 Max:    94 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 851181 in 00:04:48 = 2951.3/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary + 100838 in 00:00:30 = 3361.0/s Avg:    17 Min:     1 Max:   102 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 952019 in 00:05:18 = 2989.9/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  97703 in 00:00:30 = 3256.4/s Avg:    18 Min:     0 Max:   104 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1049722 in 00:05:48 = 3012.9/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  99602 in 00:00:30 = 3320.3/s Avg:    18 Min:     0 Max:   199 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1149324 in 00:06:18 = 3037.3/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  99271 in 00:00:30 = 3309.4/s Avg:    18 Min:     0 Max:   109 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1248595 in 00:06:48 = 3057.2/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  99568 in 00:00:30 = 3318.6/s Avg:    18 Min:     0 Max:   190 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1348163 in 00:07:18 = 3075.1/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  98396 in 00:00:30 = 3280.2/s Avg:    18 Min:     0 Max:   107 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1446559 in 00:07:48 = 3088.3/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  95239 in 00:00:30 = 3174.6/s Avg:    18 Min:     0 Max:   182 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1541798 in 00:08:18 = 3093.5/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  97676 in 00:00:30 = 3254.7/s Avg:    18 Min:     0 Max:   112 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1639474 in 00:08:48 = 3102.6/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  98031 in 00:00:30 = 3268.9/s Avg:    18 Min:     0 Max:   146 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1737505 in 00:09:18 = 3111.5/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  95085 in 00:00:30 = 3169.5/s Avg:    18 Min:     0 Max:   109 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1832590 in 00:09:48 = 3114.5/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)
summary +  38986 in 00:00:12 = 3341.0/s Avg:    17 Min:     0 Max:   108 Err:     0 (0.00%) Active: 0 Started: 60 Finished: 60
summary = 1871576 in 00:10:00 = 3118.9/s Avg:    18 Min:     0 Max:   265 Err:     0 (0.00%)

```

**Cache without cleanup task**

```
summary +      1 in 00:00:00 =    4.0/s Avg:    41 Min:    41 Max:    41 Err:     0 (0.00%) Active: 1 Started: 1 Finished: 0
summary +  24333 in 00:00:28 =  878.3/s Avg:    16 Min:     1 Max:    94 Err:     0 (0.00%) Active: 28 Started: 28 Finished: 0
summary =  24334 in 00:00:28 =  870.6/s Avg:    16 Min:     1 Max:    94 Err:     0 (0.00%)
summary +  49748 in 00:00:30 = 1659.4/s Avg:    26 Min:     0 Max:   181 Err:     0 (0.00%) Active: 58 Started: 58 Finished: 0
summary =  74082 in 00:00:58 = 1278.8/s Avg:    22 Min:     0 Max:   181 Err:     0 (0.00%)
summary +  87998 in 00:00:30 = 2934.0/s Avg:    20 Min:     0 Max:   132 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 162080 in 00:01:28 = 1843.4/s Avg:    21 Min:     0 Max:   181 Err:     0 (0.00%)
summary +  92023 in 00:00:30 = 3067.1/s Avg:    19 Min:     0 Max:   133 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 254103 in 00:01:58 = 2154.7/s Avg:    20 Min:     0 Max:   181 Err:     0 (0.00%)
summary +  93748 in 00:00:30 = 3121.1/s Avg:    19 Min:     0 Max:   159 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 347851 in 00:02:28 = 2350.9/s Avg:    20 Min:     0 Max:   181 Err:     0 (0.00%)
summary +  85934 in 00:00:30 = 2866.4/s Avg:    20 Min:     0 Max:   197 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 433785 in 00:02:58 = 2437.8/s Avg:    20 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  92181 in 00:00:30 = 3074.7/s Avg:    19 Min:     0 Max:   168 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 525966 in 00:03:28 = 2529.6/s Avg:    20 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  97157 in 00:00:30 = 3238.6/s Avg:    18 Min:     0 Max:    95 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 623123 in 00:03:58 = 2619.0/s Avg:    20 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  95466 in 00:00:30 = 3182.2/s Avg:    18 Min:     0 Max:   183 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 718589 in 00:04:28 = 2682.1/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  95374 in 00:00:30 = 3179.1/s Avg:    18 Min:     0 Max:   120 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 813963 in 00:04:58 = 2732.1/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  99808 in 00:00:30 = 3326.8/s Avg:    18 Min:     0 Max:   108 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 913771 in 00:05:28 = 2786.5/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  99764 in 00:00:30 = 3325.0/s Avg:    18 Min:     0 Max:    99 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1013535 in 00:05:58 = 2831.7/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  90864 in 00:00:30 = 3029.3/s Avg:    19 Min:     0 Max:   109 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1104399 in 00:06:28 = 2846.9/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  95857 in 00:00:30 = 3195.2/s Avg:    18 Min:     0 Max:   143 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1200256 in 00:06:58 = 2871.9/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  99893 in 00:00:30 = 3329.8/s Avg:    18 Min:     0 Max:   104 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1300149 in 00:07:28 = 2902.6/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary +  88828 in 00:00:30 = 2960.5/s Avg:    20 Min:     0 Max:   123 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1388977 in 00:07:58 = 2906.2/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary + 104680 in 00:00:30 = 3489.8/s Avg:    17 Min:     0 Max:   110 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1493657 in 00:08:28 = 2940.7/s Avg:    19 Min:     0 Max:   197 Err:     0 (0.00%)
summary + 112026 in 00:00:30 = 3734.2/s Avg:    16 Min:     0 Max:   113 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1605683 in 00:08:58 = 2985.0/s Avg:    18 Min:     0 Max:   197 Err:     0 (0.00%)
summary + 113430 in 00:00:30 = 3781.0/s Avg:    15 Min:     0 Max:    99 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1719113 in 00:09:28 = 3027.0/s Avg:    18 Min:     0 Max:   197 Err:     0 (0.00%)
summary + 100514 in 00:00:30 = 3350.5/s Avg:    17 Min:     0 Max:   127 Err:     0 (0.00%) Active: 60 Started: 60 Finished: 0
summary = 1819627 in 00:09:58 = 3043.2/s Avg:    18 Min:     0 Max:   197 Err:     0 (0.00%)
summary +   7821 in 00:00:02 = 3654.7/s Avg:    16 Min:     1 Max:    82 Err:     0 (0.00%) Active: 0 Started: 60 Finished: 60
summary = 1827448 in 00:10:00 = 3045.4/s Avg:    18 Min:     0 Max:   197 Err:     0 (0.00%)


```
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: load-tests/cache-with-cleanup-task/src/Config.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+dbHost = "postgres-service"
+dbUsername = "postgres"
+dbPassword = "postgres"
+dbName = "postgres"
+dbPort = 5432
+# Change the cache configurations if need.
+capacity = 100
+evictionFactor = 0.2
+defaultMaxAge = 120.0
+cleanupInterval = 5.0
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/Config.toml:10]:
```suggestion
cleanupInterval = 5.0

```

File: load-tests/cache-with-cleanup-task/src/client.bal [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+    public isolated function getDetail(int id) returns string|error {
+        any|error data = self.cache.get(id.toString());
+        if data !is error {
+            return data.toString();
+        }
+        stream<record {}, error?> resultStream = self.dbClient->query(
+                                                                    `SELECT * FROM Customers WHERE customerId = ${id}`);
+        any result = check resultStream.next();
+        error? err = self.cache.put(id.toString(), result);
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/client.bal:50]:
```suggestion
        check self.cache.put(id.toString(), result);
```

File: load-tests/cache-with-cleanup-task/src/client.bal [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        _ = check self.dbClient->execute(`
+            CREATE TABLE Customers (
+                customerId SERIAL,
+                customerName VARCHAR(300)
+            );
+        `);
+    }
+
+    isolated function put(int id) returns error? {
+        stream<record {}, error?> resultStream = self.dbClient->query(`SELECT * FROM Customers WHERE customerId = ${id}`);
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/client.bal:84]: Would it be better to use `queryRow()` here

File: load-tests/cache-with-cleanup-task/src/client.bal [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        any output = check resultStream.next();
+        error? err = self.cache.put(id.toString(), output);
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/client.bal:86]:
```suggestion
        check self.cache.put(id.toString(), output);
```

File: load-tests/cache-with-cleanup-task/src/service.bal [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+type Customer record {
+    int customerId;
+    string customerName;
+};
+
+cache:CacheConfig cacheConfig = {
+    capacity: capacity,
+    evictionFactor: evictionFactor,
+    defaultMaxAge: defaultMaxAge
+    //cleanupInterval: cleanupInterval
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/service.bal:39]:
```suggestion
```

File: load-tests/cache-with-cleanup-task/src/service.bal [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+};
+
+final Client cacheClient = check new ("jdbc:postgresql://" +  dbHost + "/" + dbName, dbUsername,
+                                      dbPassword, cacheConfig);
+
+public function main() returns error? {
+
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/service.bal:46]:
```suggestion
```

File: load-tests/cache-with-cleanup-task/src/client.bal [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+        }
+        return id;
+    }
+
+    public isolated function getDetail(int id) returns string|error {
+        any|error data = self.cache.get(id.toString());
+        if data !is error {
+            return data.toString();
+        }
+        stream<record {}, error?> resultStream = self.dbClient->query(
```
Comment (kaneeldias) [inline load-tests/cache-with-cleanup-task/src/client.bal:47]: Better to use `queryRow()`?

File: load-tests/cache/deployment/cache.yaml [hunk 8]
```diff
@@ -0,0 +1,90 @@
+---
```
Comment (kaneeldias) [inline load-tests/cache/deployment/cache.yaml:1]: I think this file isn't supposed to be committed

File: load-tests/cache/deployment/final.yaml [hunk 9]
```diff
@@ -0,0 +1,191 @@
+apiVersion: v1
```
Comment (kaneeldias) [inline load-tests/cache/deployment/final.yaml:1]: This is an auto-generated file through `kustomize` if I recall correctly. Not supposed to be committed.

File: load-tests/cache-with-cleanup-task/src/client.bal (hunk 2 above)
Comment (kalaiyarasiganeshalingam) [reply load-tests/cache-with-cleanup-task/src/client.bal]: This error is getting when inserting the record into the cache. This is a part of the process but the process is not dependent on this. So we don't need to return the error.

File: load-tests/cache-with-cleanup-task/src/client.bal (hunk 4 above)
Comment (kalaiyarasiganeshalingam) [reply load-tests/cache-with-cleanup-task/src/client.bal]: Same here as well.

**Review by kaneeldias — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**